app/grandchallenge/cases/tasks.py

```diff
@@ -387,8 +378,8 @@ def build_images(upload_session_uuid: UUID):
             upload_session.status = upload_session.FAILURE
             upload_session.save()
             return
-        except Exception:
-            upload_session.error_message = "An unknown error occurred"
+        except Exception as e:
+            upload_session.error_message = f"An unknown error occurred {e}"
```

Comment:
```suggestion
        except Exception:
            upload_session.error_message = f"An unknown error occurred"
```

Undo this, the exception could contain private data that is then emailed to the user. The raise below will mean that we get the traceback in sentry.